src/tools/generate/web-vitals.js

```diff
@@ -0,0 +1,30 @@
+import { getCLS, getFID, getLCP, getFCP, getTTFB } from 'web-vitals';
+
+function sendToGoogleAnalytics(metric) {
+  let event_value = 0;
+  switch (metric.name) {
+    case 'CLS':
+      // For CLS event value is first multiplied by 1000 for greater precision.
+      event_value = metric.delta * 1000;
+      break;
+    case 'TTFB':
+      // For TTFB event value excludes DNS lookup, connection negotiation, network latency, and unloading the previous document.
+      event_value = metric.value - metric.entries[0].requestStart;
```

Comment:
For TTFB we do want network setup times (DNS, TCP, etc) so this `case` could be removed. (`value` and `delta` are equivalent for TTFB)